### PR TITLE
added setBuffer to getBuffer example

### PR DESCRIPTION
**Pull request by pettaa123**

### CODE REVIEW

**Comment by blowekamp** [comment]
Very nice, thank you for the contribution.

Please remove the old `ImageGetBuffer.cs` file that this example replaces.

Also, looks like we have a configuration error here:
https://open.cdash.org/viewBuildError.php?buildid=5945378

**Comment by blowekamp** [comment]
This looks like a great contribution. Are you able to fix the build errors?

File: Examples/CSharp/ImageGetSetBuffer.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                return;
+            }
+
+            string inputFilename = args[0];
+            string outputFilename = args[1];
+
+            // Read input image
+            SitkImage input = SimpleITK.ReadImage(inputFilename);
+
+            // Cast to we know the the pixel type
```
Comment (zivy) [inline Examples/CSharp/ImageGetSetBuffer.cs:46]: "Cast to..." -> "Cast so..."

**Review by zivy — APPROVED** [review]
Probably should have been a "git mv" and then edited instead of "git rm" and creating a new file, but not critical.

**Pull request merged**